CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # App Center SDK for Unity Change Log
 
+## Release 2.4.0
+
+Updated native SDK versions:
+
+* iOS from 2.3.0 to [2.4.0](https://github.com/Microsoft/AppCenter-SDK-Apple/releases/tag/2.4.0)
+
+### App Center Auth
+
+#### iOS
+
+* **[Fix]** Redirect URIs are now hidden in logs.
```

Comment:
This version introduces Auth. We don't need to mention Feature/Fix here. Please refer to RN CHANGELOG. We are introducing Data in RN so you should get an idea for this CHANGELOG there.

Reply:
But Unity uses DotNet nuget packages, not RN's.

Reply:
What Jae meant here, it's the first release of Auth for Unity. The first release does not "fix" anything.

The data was just an example.

Here is the better example you need:

https://github.com/microsoft/appcenter-sdk-android/blob/develop/CHANGELOG.md#appcenterauth

Reply:
@guperrot Thanks for your explanation.  We push in the new commit. Could you help to review this again?